### PR TITLE
Adding eCloud public doc

### PR DESCRIPTION
**Pull request by Messier71**
Adding in doc for VM delete, thats it for me today after this

### CODE REVIEW

File: source/cloud/public/marketplace.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# eCloud Public VM Marketplace
+
+Specific OS customisation is available via the [eCloud marketplace](https://my.ukfast.co.uk/ecloud-public/marketplace). From you can chose a specific template and launch a VM pre-configured for this by selecting the template then launch appliance as shown below:
```
Comment (CountBacula) [inline source/cloud/public/marketplace.md]: missing word

File: source/cloud/public/powerCycleVM.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# eCloud Public VM power cycle
+
+In the event that you needed to reboot or power on or off a VM this can be actioned through [MyUKFast](https://my.ukfast.co.uk/ecloud-public), by selecting the VM and navigating to the reboot tab along the top. This will display the following.
```
Comment (CountBacula) [inline source/cloud/public/powerCycleVM.md]: needed - need or change tense etc.

File: source/cloud/public/marketplace.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# eCloud Public VM Marketplace
+
+Specific OS customisation is available via the [eCloud marketplace](https://my.ukfast.co.uk/ecloud-public/marketplace). From you can chose a specific template and launch a VM pre-configured for this by selecting the template then launch appliance as shown below:
+
+![launchAppliance](files/launchAppliance.png)
+
+From here you will be taken to the below device configuration page where vCPU, RAM, Disk space and other attributes of the VM can be specified. You may also be asked for unqiue information to that template as in the below example a MySQL root password defined.
+
+![applianceConfig](files/applianceConfig.png)
```
Comment (CountBacula) [inline source/cloud/public/marketplace.md]: files for both applianceconfig.png and applianceConfig.png - are they both needed?

**Review by CountBacula — CHANGES_REQUESTED** [review]
Some minor changes and potential duplicate image.

File: source/cloud/public/marketplace.md (hunk 1 above)
Comment (Messier71) [reply source/cloud/public/marketplace.md]: Added

File: source/cloud/public/marketplace.md (hunk 3 above)
Comment (Messier71) [reply source/cloud/public/marketplace.md]: Removed file not needed

File: source/cloud/public/powerCycleVM.md (hunk 2 above)
Comment (Messier71) [reply source/cloud/public/powerCycleVM.md]: Changed to need

**Comment by CountBacula** [comment]
Meta sections need to be preceeded by ` ```eval_rst`, e.g., https://github.com/ukfast/docs.ukfast.co.uk/blob/master/source/desktop/sharedexchange/sharedexdns.md

**Comment by Messier71** [comment]
Added ```eval_rst to all files

**Review by CountBacula — APPROVED** [review]

**Pull request merged**